dataprofiler/labelers/base_data_labeler.py

```diff
@@ -552,9 +586,13 @@ def _load_postprocessor(self, processor_class, dirpath):
                     str(processor_class)
                 )
             )
-        self.set_postprocessor(processor_class.load_from_disk(dirpath))
+        self.set_postprocessor(
+            cast(data_processing.BaseDataProcessor, processor_class).load_from_disk(
```

Comment:
Is the problem `processor_class` could still be a string at this point according to mypy despite the `if`

Reply:
yes, I'm not sure why mypy can't detect it